test/Microsoft.AspNet.Mvc.Core.Test/ResponseCacheTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.PipelineCore.Collections;
+using Microsoft.AspNet.Routing;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public class ResponseCacheTests
+    {
+        public static IEnumerable<object[]> CacheControlData
+        {
+            get
+            {
+                yield return new object[] { new ResponseCache { NoStore = true }, "no-store" };
+                // If no-store is set, then location cannot be set.
```

Comment:
I am verifying the exact same scenario. If it doesn't set duration or location the result will contain ONLY `no-store` which is being verified in the following 2 tests. You want it to be a separate test for readability?